sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStore.scala

```diff
@@ -396,6 +396,12 @@ object StateStoreProvider {
    */
   def create(providerClassName: String): StateStoreProvider = {
     val providerClass = Utils.classForName(providerClassName)
+    if (!classOf[StateStoreProvider].isAssignableFrom(providerClass)) {
+      throw new SparkException(
+        errorClass = "INVALID_STATE_STORE_PROVIDER",
```

Comment:
Lets use the `STATE_STORE_` prefix here

so maybe rename as `STATE_STORE_INVALID_PROVIDER`

Reply:
Sure.